File: algoritmos/03 - Estrutura condicional/Questões/questao05.py
# Questao 05

salario = float(input("Digite o salário: "))
inss = 0.0
imposto = 0.0

if salario >= 2000.00:
    imposto = 0.15
    inss = 0.10
else:
    imposto = 0.0
    inss = 0.0

print(f"Salário: R${salario:.2f} \nImposto de renda: R${salario*imposto:.2f} \nINSS: R${salario*inss:.2f}")

File: orientado a objetos/00-entrada-de-dados/exercicios-entrada-de-dados/src/Questao05.java
import java.util.Scanner;

public class Questao05 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        double salario, imposto;
        System.out.print("Digite o salário: ");
        salario = sc.nextDouble();
        imposto = salario * 0.1;

        System.out.printf("Salário: R$%.2f\n", salario);
        System.out.printf("Imposto: R$%.2f\n", imposto);
    }
}
